roles/keystone/tasks/main.yml

```diff
@@ -67,27 +67,39 @@
   template: src=etc/apache2/sites-available/keystone.conf
             dest=/etc/apache2/sites-available/keystone.conf
   notify:
-    - reload apache  
-  tags: keystone-federation
-
+    - reload apache
+  tags:
+    - keystone-federation
+    - k2k-federation
+    
 - name: configure keystone
   template: src={{ item }} dest=/etc/keystone/
             mode={{ 0644 if 'policy.json' in item else 0640 }}
             owner=keystone group=keystone
   with_fileglob: ../templates/etc/keystone/*
   notify:
     - restart keystone services
+  tags:
+    - k2k-federation
```

Comment:
I will remove this tag once I'm done testing out stuff. I was using it to only run my code.